packages/orbit-components/src/ThemeProvider/index.jsx

```diff
@@ -4,25 +4,25 @@ import { ThemeProvider as StyledThemeProvider } from "styled-components";
 import { UIDReset, UIDFork } from "react-uid";
 
 import Dictionary from "../Dictionary";
-import { QueryContextProvider } from "./QueryContext";
+import QueryContextProvider from "./QueryContext/Provider";
 
 import type { Props } from ".";
 
 const ThemeProvider = ({ theme, dictionary, children }: Props): React.Node => {
   return (
-    <QueryContextProvider>
-      <UIDReset>
-        <UIDFork>
-          <StyledThemeProvider theme={theme}>
+    <UIDReset>
+      <UIDFork>
+        <StyledThemeProvider theme={theme}>
+          <QueryContextProvider>
```

Comment:
I moved the query provider inside theme provider so that it has access to `theme`, before it was always using the default theme, oops!